.vscode/settings.json

```diff
@@ -19,5 +17,6 @@
     "editor.formatOnPaste": false,
     "editor.suggest.insertMode": "replace"
   },
-  "files.insertFinalNewline": true
-}
\ No newline at end of file
+  "files.insertFinalNewline": true,
+  "prettier.requireConfig": true
```

Comment:
Is it really necessary to modify the vscode settings?

Reply:
Prettier was used by 70% of the devs in 2020 (https://prettier.io/en/users/). If somebody has a prettier plugin, the changed files will be reformatted automatically. With requireConfig: true option, this can be turned off, until you decide to add a .prettierc config to the project. Then the formatting will work based on this file.

Until it's not defined, I think it's wise, to 'prevent' people from applying such accidental reformatting.